prototypes/form-wizard/request.html

```diff
@@ -27,8 +27,8 @@ <h3 class="usa-alert-heading">Alert</h3>
 <h4>1. Identify the federal agency that has the records you want.</h4>
 <form action="#">
   <fieldset>
-    <label for="agency" class="usa-input-required">Agency name</label>
-    <input id="agency" type="text">
+    <label for="agency" class="usa-input-required"></label>
```

Comment:
In general, I think we should keep the label, 1. for accessibility, 2. some people start typing, but then forget what the label is. They have to delete the text to make sure they're typing the right thing.